tests/components/application-connector/docs/compass-runtime-agent-tests.md

```diff
@@ -0,0 +1,135 @@
+
+Pipelines build the Compass Runtime Agent test using the **release** target from the `Makefile`.
+
+To build **and push** the Docker images of the tests, run:
+
+``` sh
+./scripts/local-build.sh {DOCKER_TAG} {DOCKER_PUSH_REPOSITORY}
+```
+
+This will build the following images:
+- `{DOCKER_PUSH_REPOSITORY}/compass-runtime-agent-test:{DOCKER_TAG}`
+
+## Running
+
+Tests can be run on any Kyma cluster with Compass Runtime Agent.
+
+Pipelines run the tests using the **test-compass-runtime-agent** target from the `Makefile`.
+
+### Deploy a Kyma cluster locally
+
+1. Provision a local Kubernetes cluster with k3d:
+   ```sh
+   kyma provision k3d
+   ```
+
+2. Install the minimal set of components required to run Compass Runtime Agent **for Kyma SKR (Compass mode)**:
+
+    ```bash
+    kyma deploy --components-file ./resources/installation-config/mini-kyma-skr.yaml --value global.disableLegacyConnectivity=true
+    ```
+
+   >**TIP:** Read more about Kyma installation in the [official Kyma documentation](https://kyma-project.io/docs/kyma/latest/02-get-started/01-quick-install/#install-kyma).
+
+### Test setup - Compass Runtime Agent configuration
+
+[The values.yaml](../resources/charts/compass-runtime-agent-test/values.yaml) file contains environment variables that are used in Compass tests. These values can be modified as needed.
+
+- `APP_DIRECTOR_URL` - Compass Director URL
+- `APP_TESTING_TENANT` - Tenant used in GraphQL calls
+- `APP_SKIP_DIRECTOR_CERT_VERIFICATION` - Skip certificate verification on the Director side
+- `APP_OAUTH_CREDENTIALS_SECRET_NAME` - Secret name for Compass OAuth credentials
+- `APP_OAUTH_CREDENTIALS_NAMESPACE` - Namespace for Compass OAuth credentials
+
+### Run the tests
+
+1. Before running the test export the following environment variables
+- `COMPASS_HOST` - host running Compass 
+- `COMPASS_CLIENT_ID` - client id used for fetching authorisation tokens
+- `COMPASS_CLIENT_SECRET` - client secret used for fetching authorisation tokens 
+
+2. To start the tests, run:
+
+``` sh
+make test-compass-runtime-agent
+```
+
+By default, the tests clean up after themselves, removing all the previously created resources and the `test` Namespace.
+
+> **CAUTION:** If the names of your existing resources are the same as the names used in the tests, running this command overrides or removes the existing resources.
+
+## Debugging
+
+### Running without cleanup
+
+To run the tests without removing all the created resources afterwards, run them in the debugging mode.
+
+1. To start the tests in the debugging mode, run:
+
+   ``` shell
+   make test-compass-runtime-agent-debug
+   ```
+
+2. Once you've finished debugging, run:
+
+   ``` shell
+   make clean-test-compass-runtime-agent-test
+   ```
+### Debugging in the IDE
+
+To run the test in your IDE you must perform the following steps.
+
+1. To prepare the cluster for debugging, run the test without cleanup:
+
+   ``` shell
+   make test-compass-runtime-agent-debug
+   ```
```

Comment:
``````suggestion
   ```bash
   make test-compass-runtime-agent-debug
   ```
``````

We use bash everywhere else for Terminal commands (at least if I review it :D but also in other places in this document), let's be consistent. Plus you had a redundant space there :)